src/haxelib/client/Main.hx

```diff
@@ -1306,7 +1307,7 @@ class Main {
 			if( p.project == prj ) {
 				if( p.version == version )
 					return;
-				throw "Library "+prj+" has two version included "+version+" and "+p.version;
+				throw "Library "+prj+" has two versions included : "+version+" and "+p.version;
```

Comment:
Nice catch for the plural :) There is no space before `:` though

Reply:
@kLabz What do you mean with `There is no space before : though`? There's a space before and after the column ` : ` same as in [L1297](https://github.com/mastef/haxelib/blob/6e09f72a82c79b7da302ec567c9948d887cecfd5/src/haxelib/client/Main.hx#L1297) which is 13 lines above it. ( And [L918](https://github.com/mastef/haxelib/blob/6e09f72a82c79b7da302ec567c9948d887cecfd5/src/haxelib/client/Main.hx#L918) )

Reply:
I meant there should not be a space before `:`, but as it's written this way all over the project (maybe because haxe has a french history :P), you can just ignore this comment :)